install/0000_30_machine-api-operator_04_metal3provisioning.crd.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: metal3provisionings.metal3.io
+spec:
+  group: metal3.io
+  names:
+    kind: Metal3Provisioning
+    listKind: Metal3ProvisioningList
+    plural: metal3provisionings
+    shortNames:
+    - m3p
```

Comment:
not a blocker, but we generally encourage people to only use shortnames for resources that are frequently gotten

Reply:
@deads2k @enxebre 
I have a few questions regarding naming.
1. As Alberto rightly pointed out, metal3provisioning.metal3.io looks odd. Better to call it, provisioning.metal3.io. So, without the API group name, the CRD would be called just **provisioning**?
2. This is a singleton CR as of now, so it is odd to call the plural "provisionings"! But, not having a value for plural is causing the unti tests to fail.
3. If we did call the CRD "provisioning", then can I keep the "shortNames" the same as the regular names? Or would you prefer "prov" ?